scripts/postbuild.sh

```diff
@@ -7,3 +7,17 @@ cp -a ../../../click/* $@
 # Build axolotl-web
 echo "update translations and build axolotl-web $@"
 cd ../../../axolotl-web&&npm run translate &&npm run build && mkdir $@/axolotl-web&&cp dist $@/axolotl-web/ -r
+
+echo $ARCH
+if [ $ARCH == "arm64" ]
+then
+    [ -z '$GOPATH' ] && cp /github/workspace/go/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_aarch64.so $@/lib/libzkgroup_linux_aarch64.so; 
+    cp $GOPATH/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_aarch64.so $@/lib/libzkgroup_linux_aarch64.so ||true; 
+    rm ${INSTALL_DIR}/\\$GITHUB_WORKSPACE||true
+fi
+if [ $ARCH == "armhf" ]
+then
+    [ -z '$GOPATH' ] && cp /github/workspace/go/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_armv7.so $@/lib/libzkgroup_linux_armv7.so; 
+    cp $GOPATH/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_armv7.so $@/lib/libzkgroup_linux_armv7.so ||true; 
+    rm ${INSTALL_DIR}/\\$GITHUB_WORKSPACE||true
```

Comment:
Instead of the ||true, we could probably use a `rm -f` here. Then we need four characters less :)

Reply:
Thanks